.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[Bug/Performance]"
+labels:
+assignees: ''
+
+---
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**System information**
+- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- HW Platform (e.g. iPhone 8, Pixel 2, Samsung Galaxy, Intel based Mac, i7 based Dell with NVIDIA GPGPU etc., or Google Cloud machine)
+- TensorFlow installed from (source or binary):
+- TensorFlow version (use command below):
+- TensorFlow Addons version:
```

Comment:
Add a bullet for how Addons was installed

Reply:
Okay